packages/router/src/grpc/view-protocol-server/delegations-by-address-index.ts

```diff
@@ -0,0 +1,116 @@
+import { Impl } from '.';
```

Comment:
This file is copied almost verbatim from the fetcher I deleted above, with a few changes that I'll highlight.